CorsixTH/Lua/hospital.lua

```diff
@@ -915,18 +942,37 @@ function Hospital:hotWarning()
     self.world.ui:playAnnouncement(announcements[math.random(1, #announcements)], AnnouncementPriority.Normal)
   end
 end
--- Called when the hospitals's boiler has broken down.
--- It will remain broken for a certain period of time.
-function Hospital:boilerBreakdown()
-  self.curr_setting = self.radiator_heat
-  self.radiator_heat = math.random(0, 1)
-  self.boiler_countdown = math.random(7, 25)
 
-  self.heating_broke = true
+--! Decide how many days the hospital functions within specification.
+--!return (int) Number of disaster-free days in the hospital.
+function Hospital:daysTillNextDisaster()
```

Comment:
I'm not a fan of "Till", I would argue "Until" is better but that's a personal preference

Reply:
Fixed